default/scripting/policies/CONFEDERATION.focs.txt

```diff
@@ -19,19 +19,20 @@ Policy
             priority = [[DEFAULT_PRIORITY]]
             effects = 
                 SetTargetInfluence value = value + 1
-
+	
         // makes all planets less stable
         EffectsGroup
-        scope = And [
-            Planet
-            OwnedBy empire = Source.Owner
-            Population low = 0.001
-        ]
-        priority = [[DEFAULT_PRIORITY]]
-        effects = [
-            SetTargetHappiness value = Value - 1
-            SetMaxTroops value = Value + 5
-        ]
+            scope = And [
+                Planet
+                OwnedBy empire = Source.Owner
+                Population low = 0.001
+            ]
+            accountinglabel = "PLC_CONFEDERATION"
```

Comment:
does this accounting label do anything different than the default without it?

Reply:
> does this accounting label do anything different than the default without it?

I realize when there is no specific accountinglabel the name is used as default.
I'll fix that.